### DISCUSSION TITLE
[GraphQL] Order random parameter

### DISCUSSION THREAD

**Discussion by lucasvallenet** [craftcms/cms#8797] [Ideas]
### Description
I'm trying to fetch some random assets with graphQL but my query always returns the same assets

### Steps to reproduce

1. Upload couple of assets
2. Generate the following graphQL query:
```
{
  assets(limit: 2, orderBy: "RAND()") {
    id
  }
}
```
3. Instead of having 2 random assets, I always receive the first 2 assets. The RAND() function doesn't seem to work

### Additional info

- Craft version: 3.4.8
- PHP version: 7.1.26
- Database driver & version: MySQL 5.7.25

Thanks !

**Comment by stephanedemotte** [comment]
+ 1

**Comment by stephanedemotte** [comment]
Any news on that ?

**Comment by andris-sevcenko** [comment]
I'll just point out that allowing to inject SQL over POST seems like a terrible idea.
